addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14912,13 +14922,13 @@ msgstr ""
 #. Description of settings category "Music -> Playback" with label #14086
 #: system/settings/settings.xml
 msgctxt "#36264"
-msgid "Category containing settings for how music playback is handled."
+msgid "This category contains the settings for how music playback is handled."
 msgstr ""
 
 #. Description of setting "Music -> Playback -> Play next song automatically" with label #489
 #: system/settings/settings.xml
 msgctxt "#36265"
-msgid "Automatically plays the next item in the current folder. For example, in Files View: After a track has been played, the next track in the same folder would automatically play."
+msgid "Automatically plays the next item in the current folder (e.g. in \"Files\" view after a track has been played, the next track in the same folder will automatically play.)"
```

Comment:
So mixing e.g. and for example then, I rather avoid mixing terms no matter if they are used in same sentence or is separate sentences.

So please confirm thats what you would like to do.

FYI, I already changed all the (e.g.) to no braces... I just haven't pushed the changes.